dev/source/docs/roadmap.rst

```diff
@@ -125,16 +123,95 @@ Hardware (PhilipR, JeffW)
 - Reference multicopter frame (PhilipR)
 - Intel Open Drone ID setup (JeffW)
 
-Logging (PeterB)
-----------------
 
-- Support adding information about messages and fields to Onboard logs
+Non-GPS Navigation, Avoidance and Path Planning (Randy / Patrick Poirier)
+-------------------------------------------------------------------------
+
+- 3D Bendy Ruler & Object Database (Randy)
+- Add support for Intel RealSense D435 (ROS & non-ROS) (Patrick, GSoC)
+- Bendy Ruler try re-implementing bendy ruler as Lua (Tridge)
+- Copter & Rover back away from objects (Randy)
+- Improve Off-Board SLAM and Object Avoidance (:ref:`dev wiki link<ros-cartographer-slam>`) (Randy, Jaime)
+- Improve reliability of T265 integration (reset handling, failsafe testing) (Randy, Jaime)
+- Seamless GPS <-> Non-GPS transitioning (Randy)
+- T265 image output to external VO (Tridge)
+
+
+Scripting (MichaelDB)
+---------------------
+
+- All mission features available in Lua (MichaelDB)
```

Comment:
Let's make the owner of this be "MichaelDB, Randy"

Reply:
ok, done.